python/cudf/cudf/tests/test_groupby.py

```diff
@@ -439,6 +439,7 @@ def test_advanced_groupby_levels():
     assert_eq(pdh, gdh)
     pdg = pdf.groupby(["x", "y", "z"]).sum()
     gdg = gdf.groupby(["x", "y", "z"]).sum()
+    assert_eq(pdg, gdg)
```

Comment:
It appears to be we were not asserting `pdg` & `dgd` at this point, Hence added an assertion.